src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/WeekNum.txt

```diff
@@ -11525,76 +11525,76 @@
 
 // **************** SAME DAY - ALL HOURS TESTCASES  ****************
 
->> WeekNum(DateTimeValue("1/7/2021 0:00:00:00"),15)
+>> WeekNum(DateTimeValue("1/7/2021 0:00:00"),15)
```

Comment:
We should not force the expression to use the enumeration (strongly-typed enum) in this function. Calls with the numeric equivalent should fail.